src/schedule/zephyr_dma_domain.c

```diff
@@ -305,7 +306,7 @@ static int zephyr_dma_domain_register(struct ll_schedule_domain *domain,
 				 dt,
 				 NULL,
 				 NULL,
-				 CONFIG_NUM_PREEMPT_PRIORITIES - 1,
+				 ZEPHYR_DMA_DOMAIN_THREAD_PRIO,
```

Comment:
why are you actually using such a low priority (a high priority number) for the DMA domain? With the Zephyr domain we use the highest available priority `-CONFIG_NUM_COOP_PRIORITIES`

Reply:
It's mostly due to the fact that its priority being in the preemptive range hasn't caused us any issues that I'm aware of. I think we can leave this in the preemptive range and switch over to cooperative range if there's ever a reason to.